src/System.Net.WebProxy/tests/WebProxyTest.cs

```diff
@@ -175,6 +175,7 @@ public static IEnumerable<object[]> BypassOnLocal_MemberData()
             yield return new object[] { new Uri($"http://{IPAddress.None}"), false };
         }
 
+        [ActiveIssue(20137, TargetFrameworkMonikers.Uap)]
```

Comment:
We discussed this with @danmosemsft and @stephentoub et. al.  The majority feel that we should use numbers, except if the number doesn't make sense in the current repo (because it is a bug in another repo).